alma/common/hpc-tuning.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Disable some unneeded services by default (administrators can re-enable if desired)
+systemctl disable firewalld
+
+# Update memory limits
+cat << EOF >> /etc/security/limits.conf
+*               hard    memlock         unlimited
+*               soft    memlock         unlimited
+*               hard    nofile          65535
+*               soft    nofile          65535
+*               hard    stack           unlimited
+*               soft    stack           unlimited
+EOF
+
+# Enable reclaim mode
+echo "vm.zone_reclaim_mode = 1" >> /etc/sysctl.conf
```

Comment:
Are the optimizations added in this PR applicable to alma? 

https://github.com/Azure/azhpc-images/pull/163

Reply:
@jithinjosepkl please comment if these optimizations are applicable to alma?

Reply:
These optimizations are applicable to Alma as per discussion with @jithinjosepkl